android-kit-base/src/main/java/com/mparticle/kits/KitConfiguration.java

```diff
@@ -610,10 +610,10 @@ boolean shouldIncludeFromAttributeValueFiltering(Map<String, String> attributes)
                 while (attIterator.hasNext()) {
                     Map.Entry<String, String> entry = attIterator.next();
                     String key = entry.getKey();
-                    int keyHash = KitUtils.hashForFiltering(key);
+                    int keyHash = HashingUtility.INSTANCE.hashFilterAvKey(key);
```

Comment:
I'm not sure if I seen this pattern before in android. Would it be possible to remove INSTANCE or at least change it to something that matches the capitalization style of the rest of the code.

Reply:
HashingUtility is an object class in Kt (similar to a class level companion object that correspond to static).
Because functions are not called using an instance the INSTANCE wouldn't be needed if we were calling if from a Kt class also, but because KitConfiguration is a java class that is mandatory and also automatic.